### ISSUE TITLE
CO2 production after pyrolysis

### ISSUE THREAD

**Issue by gforney** [firemodels/fds#686]
```
Please complete the following lines...

Application Version:5.3.0 Serial
SVN Revision Number:3193
Compile Date:Fri 30 Jan 2009
Operating System:win xp

Describe details of the issue below:

Ok mcgratta, I don't want to waste your time but I want your opinion on my 
problem . I send you a new simpler file . I wanted actually to mix solid 
and gas combustion ( with CO_PRODUCTION=.TRUE. ) , is it impossible ??? 
Please ask me because if it's the case I don't try anymore to do it and I 
would earn time...I know that 41000 is huge but in my case the external 
flux is 10cal/(cm^2.s)=40 J / (cm^2.s) = 40 W / cm^2 = 40 000 kW/m^2 !!!


I just want to see the reaction between CO and O2 after the pyrolysis, is 
it really impossible ???

I'm indeed a begginner but I want to progress so please give me your 
advices.

Thanks 



Gael
```

Original issue reported on code.google.com by `lagalz@msn.com` on 2009-04-17 08:04:25

<hr>
- _Attachment: [pyrolysis3.fds](https://storage.googleapis.com/google-code-attachments/fds-smv/issue-708/comment-0/pyrolysis3.fds)_

**Comment by gforney** [comment]
```
Here is the fundamental problem with your analysis. In FDS there are two distinct 
ways to model combustion. The first is the mixture fraction method and this is the

default. Y_O2_INFTY is a parameter that is associated with the mixture fraction 
model. It sets the background concentration of oxygen. The second way to model 
combustion is via one or more finite rate reactions. To use this approach, you 
specify explicitly all the gas species via the SPEC lines, except for the background

species, which is set on the MISC line with BACKGROUND_SPECIES. You have to also 
specify, via the REAC line, the arrhenius constants associated with the reaction. 
There is a section in the User's Guide (Section 9.3) that tells you how to do this.


You have specified parameters that are used in both approaches. This means that half

of your inputs are being ignored. At the moment, it appears that you have a 10 m 
thick block that is being heated by an external source set to 50 kW/m2. There is no

gas phase reaction taking place. CO_PRODUCTION only works with the mixture fraction

model, but the mixture fraction model does not work with specified species.

In short, you cannot do the analysis with FDS. I suggest you consider another 
modeling approach.
```

Original issue reported on code.google.com by `mcgratta` on 2009-04-17 12:01:45

**Comment by gforney** [comment]
```
Ok Kevin , thanks a lot for all these very clear explications , I didn't understand

that CO_PRODUCTION only works with the mixture fraction model , here was my main 
mistake , thanks again for having solve my problem and for having spend time on it

Gael
```

Original issue reported on code.google.com by `lagalz@msn.com` on 2009-04-17 12:53:08
